storage/container_storage_interface/persistent-storage-csi-migration.adoc

```diff
@@ -0,0 +1,30 @@
+[id="persistent-storage-csi-migration"]
+= CSI automatic migration
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-migration
+
+toc::[]
+
+== Overview
+
+{product-title} provides automatic migration for supported in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers. With automatic migration enabled, volumes provisioned using in-tree storage plug-ins supported by this feature are migrated to their counterpart CSI drivers.
+
+:FeatureName: CSI automatic migration
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+The following drivers are supported:
+
+* Amazon Web Services (AWS) Elastic Block Storage (EBS)
+
+* OpenStack Cinder
+
+CSI automatic migration should be seamless. Enabling this feature does not change how you use all existing API objects (for example, PersistentVolumes, PersistentVolumeClaims, and StorageClasses).
```

Comment:
* "CSI automatic migration should be seamless." is a little marketing-y. I'd consider removing it and just starting with "Enabling CSI automatic migration does not change...)
* the objects listed should either be lowercase like "persistent volumes" or should be the proper object in backticks "`PersistentVolume` objects".

Reply:
Going to stick with the language as it's been vetted.